src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -135,11 +138,13 @@ export class DetailsViewCommandBar extends React.Component<
     private renderCommandButtons(): JSX.Element {
         const reportExportElement: JSX.Element = this.renderExportButton();
         const startOverElement: JSX.Element = this.renderStartOverButton();
+        const saveAssessmentElement: JSX.Element | null = this.renderSaveAssessmentButton();
 
-        if (reportExportElement || startOverElement) {
+        if (reportExportElement || saveAssessmentElement || startOverElement) {
```

Comment:
It looks like we might not have test coverage for this if statement evaluating to false because `saveAssessmentElement` is never a falsy value during the tests. It might be a good idea to tweak `renderSaveAssessmentButton()` to make it easier to test

Reply:
Updated, thank you!